### PR TITLE
Create observability configuration document

### PR DESCRIPTION
**Pull request by vankichi**
### Description:

I have created the observability configuration (v1.7~).
It requires OpenTelemetry and OpenTelemtry Protocol for observing.

### Versions:

- Go Version: 1.19.2
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.8

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/main/CONTRIBUTING.md) document and completed [our CLA agreement](https://cla-assistant.io/vdaas/vald).
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?

### CODE REVIEW

File: docs/user-guides/observability-configuration.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+      # Export interval for metrics
+      metrics_export_interval: "1s"
+      # Maximum duration for exporting metrics
+      metrics_export_timeout: "1m"
+```
+
+#### gRPC Configuration
+
+The interceptor configuration is required to send the metrics and trace data related to gRPC.
+You can add the interceptor to the server-side or client-side by editing `defaults.server_config.servers.grpc.server.grpc.interceptors` parameters.
```
Comment (hlts2) [inline docs/user-guides/observability-configuration.md:209]:
```suggestion
You can add the interceptor to the server-side and client-side by editing `defaults.server_config.servers.grpc.server.grpc.interceptors` parameters.
```

**Review by hlts2 — APPROVED** [review]
LGTM

**Review by datelier — APPROVED** [review]

File: docs/user-guides/observability-configuration.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+<div class="notice">
+You can change parameters such as the export host by editing the manifest for your demands.
+</div>
+
+### Cert-Manager
+
+A `Cert-Manager` is required to enable to use of operators on the Kubernetes Cluster.
+
+```bash
+make k8s/external/cert-manager-deploy
```
Comment (hlts2) [inline docs/user-guides/observability-configuration.md:40]: Sorry, I overlooked it.
```suggestion
make k8s/external/cert-manager/deploy
```

**Review by kpango — DISMISSED** [review]

File: docs/user-guides/observability-configuration.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+    client:
+      dial_option:
+        interceptors:
+          - TraceInterceptor
+```
+
+## Monitoring telemetry data
+
+Telemetry data can be monitored using Grafana, Jaeger, etc.
+Vald publishes the sample manifest, which enables Grafana and Jaeger.
```
Comment (kevindiu) [inline docs/user-guides/observability-configuration.md:235]: where can i find the sample grafana / jaeger configuration/manifest?

**Review by datelier — DISMISSED** [review]

**Review by hlts2 — DISMISSED** [review]

**Review by kevindiu — APPROVED** [review]

**Review by hlts2 — APPROVED** [review]
LGTM!

**Review by kpango — APPROVED** [review]

**Review by datelier — APPROVED** [review]

**Pull request merged**